### PR TITLE
[fix] Upgrade node bindings for node 12

### PR DESCRIPTION
**Pull request by imothee**
Fixes errors with ToObject and GetFunction to build against Node 12.
Fixes deprecations against Get and Set for Node 12.

### CODE REVIEW

**Comment by imothee** [comment]
Not sure why Travis is having seg faults, it works on my machine (OSX), docker node-alipne and heroku. Will look into this when I get a chance.

**Comment by cymen** [comment]
I think this is some sort of caching issue -- I forked this repo and merged your PR into my fork (along with #23) and was able to run Travis successfully:

https://travis-ci.org/cymen/node-postal/builds/655077125

File: src/parser.cc [hunk 1]
```diff
@@ -98,7 +98,7 @@ void init(v8::Local<v8::Object> exports) {
         return;
     }
 
-    exports->Set(Nan::New("parse_address").ToLocalChecked(), Nan::New<v8::FunctionTemplate>(ParseAddress)->GetFunction());
+     Nan::Set(exports, Nan::New("parse_address").ToLocalChecked(), Nan::GetFunction(Nan::New<v8::FunctionTemplate>(ParseAddress)).ToLocalChecked());
```
Comment (cymen) [inline src/parser.cc]: There is an extra space character before `Nan` on this line.

**Comment by missinglink** [comment]
I can confirm that this works on OSX `node v12.16.1`

**Comment by justathoughtor2** [comment]
I can also confirm that imothee's fork is the only way I've been able to get nodepostal running on Node.js 12.x.

**Comment by missinglink** [comment]
Hi all, I've just been granted `owner` on this repo and would be happy to review, test & merge this PR.
It's been a while and the branch now has conflicts, @imothee could you please rebase against the current `master` branch?

**Comment by imothee** [comment]
@missinglink It looks like the changes I had added in this are superseded by the fixes merged in in #25 and I confirmed that it builds on node 12 and 14 so closing this in lieu of the existing changes. Thanks for taking ownership of this code :)

**Pull request closed without merging**